src/main/java/com/vamae/connect4/controller/Connect4GameController.java

```diff
@@ -1,33 +1,47 @@
 package com.vamae.connect4.controller;
 
+import com.vamae.connect4.payload.request.InitializationRequest;
+import com.vamae.connect4.payload.request.JoinRequest;
+import com.vamae.connect4.payload.request.MoveRequest;
 import com.vamae.connect4.service.Connect4GameService;
 import com.vamae.connect4.entity.Connect4Game;
 import lombok.RequiredArgsConstructor;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/connect4")
 public class Connect4GameController {
 
     private final Connect4GameService connect4GameService;
 
-    @GetMapping("/createGame")
+    @GetMapping("/games")
+    public List<Connect4Game> getAllGames() {
```

Comment:
the client does not need to know the details of the game in each lobby, it is enough for him to know only the id of the game to which he wants to join